Xamarin.Forms.Core/BorderElement.cs

```diff
@@ -1,14 +1,86 @@
-﻿namespace Xamarin.Forms
+﻿using System;
+
+namespace Xamarin.Forms
 {
 	static class BorderElement
 	{
+		const int DefaultBorderRadius = 5;
+		const int DefaultCornerRadius = -1;
+
 		public static readonly BindableProperty BorderColorProperty =
 			BindableProperty.Create("BorderColor", typeof(Color), typeof(IBorderElement), Color.Default,
 									propertyChanged: OnBorderColorPropertyChanged);
 
+		public static readonly BindableProperty BorderWidthProperty = BindableProperty.Create("BorderWidth", typeof(double), typeof(IBorderElement), -1d);
+
+		[Obsolete("BorderRadiusProperty is obsolete as of 2.5.0. Please use CornerRadius instead.")]
+		public static readonly BindableProperty BorderRadiusProperty = BindableProperty.Create("BorderRadius", typeof(int), typeof(IBorderElement), defaultValue: DefaultBorderRadius,
+			propertyChanged: BorderRadiusPropertyChanged);
+
+		public static readonly BindableProperty CornerRadiusProperty = BindableProperty.Create("CornerRadius", typeof(int), typeof(IBorderElement), defaultValue: DefaultCornerRadius,
+			propertyChanged: CornerRadiusPropertyChanged);
```

Comment:
as the property changed handler name is part of the API, we should stick to a common naming strategy. `OnCornerRadiusPropertyChanged`